README.md

```diff
@@ -235,9 +242,9 @@ $ yarn --silent beehive-flow status
     "major": 0,
     "minor": 11,
     "patch": 0,
-    "preRelease": "alpha"
+    "preRelease": "rc"
```

Comment:
I can't put this comment in the right spot because github won't let me comment on an unchanged line.

On line 206 above in the "stamp" command docs you mention the rules for converting `a.b.0-alpha` to a timestamped version but I believe we don't have that any more.

Reply:
Fixed.